reactComponents/src/searchbar/Context/TokenSearch.tsx

```diff
@@ -0,0 +1,3 @@
+import React, { createContext } from 'react'
+const TokenSearchContext = createContext<any>({})
```

Comment:
It's tricky to define the type for this context

Reply:
I think it would be much better to define the types through typescript vs using a text document - this way we have assurances that it is kept up to date